README.rst

```diff
@@ -2,116 +2,123 @@
+
+Fulltext differs from other libraries in that it tries to use file data in the
+form it is given. For most backends, a file-like object or path can be handled
+directly, removing the need to write temporary files.
+
+Fulltext uses native python libraries when possible and utilizes CLI tools
+when necessary, for example, the following CLI tools are required.
 
-Fulltext is a library that makes converting various file formats to
-plain text simple. Mostly it is a wrapper around shell tools. It will
-execute the shell program, scrape it's results and then post-process the
-results to pack as much text into as little space as possible.
+ * antiword - Legacy .doc (Word) format.
+ * unrtf - .rtf format.
+ * pdf2text (poppler-utils) - .pdf format.
 
 Supported formats
 -----------------
 
-The following formats are supported using the command line apps listed.
-
--  application/pdf: pdftotext
--  application/msword: antiword
--  application/vnd.openxmlformats-officedocument.wordprocessingml.document:
-   docx2txt
--  application/vnd.ms-excel: convertxls2csv
--  application/rtf: unrtf
--  application/vnd.oasis.opendocument.text: odt2txt
--  application/vnd.oasis.opendocument.spreadsheet: odt2txt
--  application/zip: funzip
--  application/x-tar, gzip: tar & gunzip
--  application/x-tar, bzip2: tar & bunzip2
--  application/rar: unrar
--  text/html: html2text
--  text/xml: html2text
--  image/jpeg: exiftool
--  video/mpeg: exiftool
--  audio/mpeg: exiftool
--  application/octet-stream: strings
+* csv - Uses Python ``csv`` module.
+* doc - Uses ``/bin/antiword`` CLI tool.
+* docx - Uses Python ``docx2txt`` module.
+* html - Uses Python ``BeautifulSoup`` module.
+* ods - Uses Python ``lxml``, ``zipfile`` modules.
+* odt - Uses Python ``lxml``, ``zipfile`` modules.
+* pdf - Uses ``/bin/pdf2text`` CLI tool.
+* rtf - Uses ``/bin/unrtf`` CLI tool.
+* text - Default backend that uses various Python stdlib modules to extract
+         strings from arbitrary (possibly) binary files.
+* xls - Uses Python ``xlrd`` module.
+* xlsx - Uses Python ``xlrd`` module.
+* zip - Uses Python ``zipfile`` module.
 
 Installing tools
 ----------------
 
-Fulltext uses the above command line programs to function. Therefore, it is not
-useful unless you have installed them. Many of them can be installed via your system's
-package manager. I use Fedora, thus the following command installed most of the
-required packages.
+Fulltext uses a number of pure Python libraries. Fulltext also uses the
+command line tools: antiword, pdf2text and unrtf.
 
-::
+.. code:: bash
 
-    $ sudo yum install xls2csv odt2txt antiword poppler-utils unrtf \
-    perl-Image-ExifTool html2text binutils unrar gzip bzip2 unzip
+    $ sudo yum install antiword unrtf poppler-utils
 
-The docx2txt utility is not avaialable in a package.
+Or for debian-based systems:
 
-http://docx2txt.sourceforge.net/
+.. code:: bash
+
+    $ sudo apt-get install antiword unrtf poppler-utils
```

Comment:
I also had to install `libjpeg-dev` for `Pillow`

Reply:
Hmm, I will check that pillow is even used... That may be left over. I will either remove it or write a backend for images that uses it.

Reply:
Ah, none of the above, Pillow is a requirement of a requirement, I will just add it to README.